x-pack/plugins/apm/server/lib/correlations/get_correlations_for_slow_transactions/index.ts

```diff
@@ -70,14 +72,21 @@ export async function getCorrelationsForSlowTransactions({
           query: {
             bool: {
               // foreground filters
-              filter: [
-                ...backgroundFilters,
-                {
-                  range: {
-                    [TRANSACTION_DURATION]: { gte: durationForPercentile },
+              filter: backgroundFilters,
```

Comment:
What's the reason for removing the range query for duration? Will removing it not hurt perf since more documents are retrieved?

Reply:
moving the range query to the `function_score` provided us with a score that we needed to order foreground results that were most important to us (those with higher durations)